test/src/base/MooseTestApp.C

```diff
@@ -55,6 +55,7 @@ MooseTestApp::validParams()
   params.set<bool>("automatic_automatic_scaling") = false;
   params.set<bool>("use_legacy_material_output") = false;
   params.set<bool>("use_legacy_initial_residual_evaluation_behavior") = false;
+  params.set<bool>("use_legacy_fixed_point_execute_on") = false;
```

Comment:
My goal is to pass the tests with or without this line.

Reply:
I have verified this on my local machine.

Reply:
Let's try to avoid having a legacy mode. Supporting both on & off legacy mode is not something we test properly at this point.

Reply:
You mean we will push to fix all modules and applications? That will be some works. It could be quite challenging. I was thinking that if the legacy message is too irritating, we can either not print anything or just print a light info message.

Reply:
no I mean keep the defaults correct so we dont need to fix all modules and applications